pkg/client/restconfig/config.go

```diff
@@ -40,15 +40,15 @@ func defaultGetCurrentUser() (*user.User, error) {
 	return user.Current()
 }
 
-// newConfigPath returns the correct kubeconfig file path to use, depending on
-// the current user settings and the runtime environment.
-func newConfigPath() (string, error) {
-	// First try the KUBECONFIG variable.
+// KubeConfigPath returns the path to the kubeconfig:
+// 1. ${KUBECONFIG}, if non-empty
+// 2. ${userCurrentTestHook.HomeDir}/.kube/config, if userCurrentTestHook is set
+// 3. ${HOME}/.kube/config
+func KubeConfigPath() (string, error) {
```

Comment:
Does this need to be exported?

Reply:
bah. this change snuck in but it's for the follow up. I'll remove it from this PR.

Reply:
fixed.